CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.1.0](https://github.com/voxpupuli/puppet-snmp/tree/v4.1.0) (2018-11-23)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-snmp/compare/v4.0.0...v4.1.0)
+
+**Implemented enhancements:**
+
+- Implement snmpv3\_user fact and snmp::snmpv3\_usm\_hash function [\#157](https://github.com/voxpupuli/puppet-snmp/pull/157) ([smoeding](https://github.com/smoeding))
+
+**Closed issues:**
+
+- Move the module to voxpupuli [\#84](https://github.com/voxpupuli/puppet-snmp/issues/84)
```

Comment:
This was part of the 4.0 release already. IMHO it's confusing to put it in 4.1.0 then so I'd use the `skip-changelog` label on it.

Reply:
I was confused me too. I didn't know how to manage this.

So i added the `skip-changelog` tag.